modules/entity_version_workflows/tests/Functional/EntityVersionNodeRevisionTest.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace Drupal\Tests\entity_version_workflows\Functional;
+
+use Drupal\Core\Url;
+use Drupal\Tests\BrowserTestBase;
+use Drupal\Tests\entity_version\Traits\EntityVersionAssertionsTrait;
+
+/**
+ * Tests the node revisions work with entity versions.
+ */
+class EntityVersionNodeRevisionTest extends BrowserTestBase {
+
+  use EntityVersionAssertionsTrait;
+
+  /**
+   * Admin user.
+   *
+   * @var \Drupal\Core\Session\AccountInterface
+   */
+  protected $adminUser;
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $modules = [
+    'path',
+    'node',
+    'user',
+    'system',
+    'workflows',
+    'content_moderation',
+    'entity_version',
+    'entity_version_workflows',
+    'entity_version_workflows_example',
+  ];
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp() {
+    parent::setUp();
+
+    $this->adminUser = $this->drupalCreateUser([
```

Comment:
I think it should be enough to test with just this permissions:      
` 'access administration pages',`
` 'administer nodes',`